client/ayon_core/pipeline/traits/utils.py

```diff
@@ -0,0 +1,38 @@
+"""Utility functions for traits."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from clique import assemble
+
+from ayon_core.pipeline.traits.time import FrameRanged
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+
+def get_sequence_from_files(paths: list[Path]) -> FrameRanged:
+    """Get original frame range from files.
+
+    Note that this cannot guess frame rate, so it's set to 25.
+
+    Args:
+        paths (list[Path]): List of file paths.
+
+    Returns:
+        FrameRanged: FrameRanged trait.
+
+    """
+    col = assemble([path.as_posix() for path in paths])[0][0]
```

Comment:
This will error on:

- File paths that do not have a sequence identifier
- File paths that only contain a single frame

Because both won'tbe detected as collections but as remainder